cluster/src/main/java/org/apache/iotdb/cluster/query/ClusterPlanRouter.java

```diff
@@ -504,4 +507,30 @@ private CreateMultiTimeSeriesPlan createSubPlan(CreateMultiTimeSeriesPlan plan)
     subPlan.setIndexes(new ArrayList<>());
     return subPlan;
   }
+
+  /**
+   * @param plan InsertRowsOfOneDevicePlan
+   * @return key is InsertRowsOfOneDevicePlan, value is the partition group the plan belongs to. All
+   *     InsertRowPlans in InsertRowsOfOneDevicePlan belong to one same storage group.
+   */
+  private Map<PhysicalPlan, PartitionGroup> splitAndRoutePlan(InsertRowsOfOneDevicePlan plan)
+      throws MetadataException {
+    Map<PhysicalPlan, PartitionGroup> result = new HashMap<>();
+    Map<PartitionGroup, List<InsertRowPlan>> groupPlanMap = new HashMap<>();
+    PartialPath storageGroup = getMManager().getStorageGroupPath(plan.getPrefixPath());
+    for (InsertRowPlan p : plan.getRowPlans()) {
```

Comment:
hi , Good Job.
I have a little...  if you could add some UT or IT, this will validation...

Reply:
OK, I will add some UT cases. Until now the whole IoTDB project have no IT cases running in cluster, at least I haven't found any...

Reply:
Actually, we are using testcontainer to run IT cases...
See https://github.com/apache/iotdb/blob/master/testcontainer/src/test/java/org/apache/iotdb/db/sql/Cases.java

Reply:
Good, let me have a try~